startServer.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Default values for arguments
+freshSync=false
+source=""
+
+# Parse command line arguments
+# Two arguments are expected: --freshSync and --source
+# --freshSync is a boolean argument, which when specified true, will start the server with a fresh sync from the source DB
+# --source is a string argument, which tells the script which source DB to use, either staging or prod
+while [[ $# -gt 0 ]]
+do
+  key="$1"
+
+  case $key in
+    -f|--freshSync)
```

Comment:
Why are all these things in separate lines? Makes it harder to read. It's necessary?

Reply:
This is done by linter not me :p

Reply:
Oh... bash ka bhi linter daal diye. bash is supposed to be painful :D